src/Internal/Requirements/RequirementValidator.php

```diff
@@ -39,6 +39,14 @@ public static function instance(): RequirementValidator {
 	 * @param RuntimeException $e
 	 */
 	protected function add_admin_notice( RuntimeException $e ) {
+		// Check if the plugin is active. If not, display error message and terminate.
+		if ( ! is_plugin_active( 'woocommerce-google-listings-and-ads/woocommerce-google-listings-and-ads.php' ) ) {
```

Comment:
The plugin and folder don't have the `woocommerce-` prefix. This means that we will always die with the error and never show the admin notice. There are certain cases when we already activated the plugin we want to show the admin notice.

There are also cases where it gets installed in a different folder. Often this is when a version gets added in the folder name. We can probably have a look how that's done in other extensions but otherwise something along the lines of setting a constant in the main plugin file like:
```php
define( 'WC_GLA_INSTALLED_DIR', basename( __DIR__ ) );
```

And then using that here:
```suggestion
		if ( ! is_plugin_active( trailingslashit( WC_GLA_INSTALLED_DIR ) . 'google-listings-and-ads.php' ) ) {
```

You could probably also pull in PluginHelper since that's a trait, and then add a helper function to get the active plugin string (get_main_filename is already there).

Reply:
Thanks for picking this up. I was being completely daft on this. I have used the PluginHelper trait.